packages/cli/src/service/job.ts

```diff
@@ -0,0 +1,135 @@
+import { join, isAbsolute } from 'path';
+import * as fs from 'fs-extra';
+import { PipelineStatus } from '@pipcook/pipcook-core';
+import { JobStatusValue, JobResp } from '@pipcook/sdk';
+import { getFile } from '../utils/request';
+import { installPackageFromConfig } from './pipeline';
+import { tunaMirrorURI } from '../config';
+import { logger, initClient, traceLogger, extractToPath, parseConfigFilename, streamToJson } from '../utils/common';
+
+export async function list(opts: any): Promise<void> {
+  const client = initClient(opts.host, opts.port);
+  const jobs = await client.job.list();
+  if (jobs.length > 0) {
+    console.table(jobs.map((job) => {
+      return { ...job, status: JobStatusValue[job.status] };
+    }), [ 'id', 'status', 'evaluatePass', 'createdAt' ]);
+  } else {
+    console.info('no job is created.');
+  }
+}
+
+export async function remove(id: string, opts: any): Promise<void> {
+  const client = initClient(opts.host, opts.port);
+  logger.start('removing jobs...');
+  try {
+    if (id === 'all') {
+      id = undefined;
+    }
+    await client.job.remove(id);
+    logger.success('remove jobs successfully');
+  } catch (err) {
+    logger.fail(err.message);
+  }
+}
+
+export async function stop(id: string, opts: any): Promise<void> {
+  const client = initClient(opts.host, opts.port);
+  logger.start('stopping jobs...');
+  try {
+    await client.job.cancel(id);
+    logger.success('stop job successfully');
+  } catch (err) {
+    logger.fail(err.message);
+  }
+}
+
+export async function log(id: string, opts: any): Promise<void> {
+  const client = initClient(opts.host, opts.port);
+  try {
+    const log = await client.job.log(id);
+    console.log(log);
+  } catch (err) {
+    console.error(err.message);
```

Comment:
suggest using logger object for all console-related outputs

Reply:
Done.